src/jwtcookie/Authentication/JwtCookie/JwtCookieHandler.cs

```diff
@@ -19,20 +22,30 @@ namespace AltinnCore.Authentication.JwtCookie
     /// </summary>
     public class JwtCookieHandler : AuthenticationHandler<JwtCookieOptions>
     {
+        private JwtSecurityTokenHandler _validator = new JwtSecurityTokenHandler();
+        private readonly OidcProviderSettings _oidcProviderSettings;
+        private readonly PlatformSettings _platformSettings;
+
         /// <summary>
         /// The default constructor
         /// </summary>
         /// <param name="options">The options</param>
+        /// <param name="oidcProviderSettings">The settings related to oidc providers</param>
+        /// <param name="platformSettings">Platform specific settings</param>
         /// <param name="logger">The logger</param>
         /// <param name="encoder">The Url encoder</param>
         /// <param name="clock">The system clock</param>
         public JwtCookieHandler(
             IOptionsMonitor<JwtCookieOptions> options,
+            IOptions<OidcProviderSettings> oidcProviderSettings,
+            IOptions<PlatformSettings> platformSettings,
             ILoggerFactory logger,
             UrlEncoder encoder,
             ISystemClock clock)
             : base(options, logger, encoder, clock)
         {
+            _oidcProviderSettings = oidcProviderSettings.Value;
+            _platformSettings = platformSettings.Value;
```

Comment:
I know I have thought about platform settings before I remembered it was configured directly. Do we really need it now?